elf_reader.go

```diff
@@ -102,6 +107,12 @@ func LoadCollectionSpecFromReader(rd io.ReaderAt) (*CollectionSpec, error) {
 	if err != nil {
 		return nil, xerrors.Errorf("load version: %w", err)
 	}
+	if ec.version == useCurrentKernelVersion {
+		ec.version, err = CurrentKernelVersion()
```

Comment:
This causes a build error on non-linux platforms because there is no `CurrentKernelVersion` defined.

Reply:
Added a `kernel_version_unsupported.go` file